packages/mco/constants/disaster-recovery.ts

```diff
@@ -130,3 +130,8 @@ export const PROTECTED_VMS = 'PROTECTED_VMS';
 export const K8S_RESOURCE_SELECTOR_LABEL_KEY =
   'ramendr.openshift.io/k8s-resource-selector';
 export const STORAGE_ID_LABEL_KEY = 'ramendr.openshift.io/storageid';
+
+export enum BackendType {
+  DataFoundation = 'Data Foundation',
+  NonDataFoundation = 'Third Party Storage',
```

Comment:
What about calling it `ThirdParty` or `Foreign` or similar? Calling it `Non...` it's a catch-all: hard to add any other option as theoretically it's already caught by `NonDF` (dual option: DF/non-DF):
```suggestion
  ThirdParty = 'Third Party Storage',
```

Reply:
@vbnrh What I mean its not recommended to make things dual, like: Black/non-Black: it's meaningless: we want to know what it is, not what it's not.